\hypertarget{classpip_1_1__internal_1_1exceptions_1_1InstallationError}{}\doxysection{pip.\+\_\+internal.\+exceptions.\+Installation\+Error Class Reference}
\label{classpip_1_1__internal_1_1exceptions_1_1InstallationError}\index{pip.\_internal.exceptions.InstallationError@{pip.\_internal.exceptions.InstallationError}}


Inheritance diagram for pip.\+\_\+internal.\+exceptions.\+Installation\+Error\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=350pt]{classpip_1_1__internal_1_1exceptions_1_1InstallationError__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for pip.\+\_\+internal.\+exceptions.\+Installation\+Error\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=217pt]{classpip_1_1__internal_1_1exceptions_1_1InstallationError__coll__graph}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}General exception during installation\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
inventaria\+\_\+env/lib/python3.\+10/site-\/packages/pip/\+\_\+internal/exceptions.\+py\end{DoxyCompactItemize}
